source/Calamari.AzureWebApp/ServicePrincipal.cs

```diff
@@ -1,17 +1,27 @@
 ﻿using System.Threading.Tasks;
-using Microsoft.IdentityModel.Clients.ActiveDirectory;
-﻿using Calamari.Common.Plumbing.Logging;
+using Calamari.Common.Plumbing.Logging;
+using Microsoft.Identity.Client;
 
 namespace Calamari.AzureWebApp
 {
     static class ServicePrincipal
     {
         public static async Task<string> GetAuthorizationToken(string tenantId, string applicationId, string password, string managementEndPoint, string activeDirectoryEndPoint)
-        {
+        { 
             var authContext = GetContextUri(activeDirectoryEndPoint, tenantId);
```

Comment:
Any recommended reading around ADAL and MSAL? I've had a look at https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-net-differences-adal-net and as far as I understand from our implementation AAD is used in this initial authentication flow but MSAL tokens are used by the client to authenticate there on out?